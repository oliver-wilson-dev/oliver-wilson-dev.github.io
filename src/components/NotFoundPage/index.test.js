import React from 'react';
import { shallow } from 'enzyme';
import About from '.';

jest.mock('react-router-dom', () => ({
  Link: () => null
}));

jest.mock('./search.svg', () => {
  const SearchIcon = () => null;

  return SearchIcon;
});

jest.mock('../../containers/Page', () => {
  const SubPage = ({ children }) => <div>{children}</div>;

  return SubPage;
});

const render = () => shallow(<About />);

describe('About', () => {
  it('should render correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
