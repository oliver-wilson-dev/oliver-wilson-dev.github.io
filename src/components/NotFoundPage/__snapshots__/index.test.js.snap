// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render correctly 1`] = `
<SubPage
  additionalStyles="notFound"
>
  <h1
    className="pageHeader"
  >
    Page not found
  </h1>
  <SearchIcon
    className="searchIcon"
  />
  <Link
    className="homeLink"
    to="test-home-path"
  >
    Return to Home Page
  </Link>
</SubPage>
`;
